docs/user/dql/expressions.rst

```diff
@@ -160,7 +160,7 @@ Here is an example for different type of comparison operators::
     +---------+----------+---------+----------+----------+---------+
 
 It is possible to compare datetimes. When comparing different datetime types, for example `DATE` and `TIME`, both converted to `DATETIME`.
-The following rule is applied on coversion: a `TIME` applied to today's date; `DATE` is interpreted at midnight.
+The following rule is applied on coversion: a `TIME` applied to today's date; `DATE` is interpreted at midnight.:
```

Comment:
```suggestion
The following rule is applied on coversion: a `TIME` applied to today's date; `DATE` is interpreted at midnight::
```

Following the conventions of the rest of the documentation I believe that this should be a `::` at the end rather than `.:`

Reply:
Apologies, didn't mean to open this PR against OpenSearch.